src/routes/list/list.ts

```diff
@@ -0,0 +1,95 @@
+import * as express from 'express'
+
+import { ListFacade } from '../../bl/listFacade'
+
+export class List {
+    static async getAllFromBoardId(req: express.Request, res: express.Response) {
+        try {
+            var boardId = req.params.board_id
+            if (boardId == null) {
+                res.status(400).json({ message: 'Request parameter invalide'}) 
+            } else {
+                // else if (forbidden : if the user appratient à la team alors c'est bon) res.status(403)
+                // else if (Method Not Allowed) res.status(405)
+                const lists = await ListFacade.getAllFromBoardId(req.params.board_id)
+                res.status(200).json(lists)
+            }
+        } catch (e) {
+            res.status(404).json({ message: e.message})
+        }
+    }
+
+    static async getOneById(req: express.Request, res: express.Response) {
+        try {
+            var listId = req.params.list_id
+            if (listId == null) {
```

Comment:
No need for a var here, you can place `req.params.list_id` in the if statement

Reply:
And in case you need to store something, please use `const`.